apps/website/src/pages/docs/datepicker.mdx

```diff
@@ -14,44 +14,74 @@ import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-datepicker' />
```

Comment:
Since this `Placeholder` was removed, the `Placeholder` import statement can also be removed

Reply:
removed